src/main/java/com/conversions/HexadecimalToBinary.java

```diff
@@ -0,0 +1,48 @@
+package com.conversions;
+
+public class HexadecimalToBinary {
+	 /**
```

Comment:
Your code fails in case of floating point conversion, 'ABC.BC' try doing it for real values also

Reply:
Good work but you missed out in the 2nd file.